Automattic/pocket-casts-ios
# Playback 2024: Longest Episode

| 📘 Part of: #2250 |
|:---:|

Part of #2346

| | |
| -- | -- |
| <video src="https://github.com/user-attachments/assets/2d8b04f7-fa50-4fbd-b183-908d5b88b1e3"> | <video src="https://github.com/user-attachments/assets/95c4d00d-14b9-4463-8780-173bd528619b"> |

## To test

* Enable the `endOfYear2024` feature flag
* Navigate through the stories until you hit this Longest Episode screen
* Ensure layout looks alright across devices

## Checklist
